config.rb

```diff
@@ -54,6 +54,9 @@ def self.test?
   optional :omniauth_github_secret, string, clear: true
   optional :omniauth_google_id, string, clear: true
   optional :omniauth_google_secret, string, clear: true
+  optional :hetzner_ssh_private_key, string, clear: true
+  optional :hetzner_ssh_private_key_passphrase, string, clear: true
+  optional :operational_public_keys, string
```

Comment:
Maybe rename this to `operator_ssh_public_keys`?

Reply:
Yes this reads better, done